### PR TITLE
Capstone Project Review

### PR DESCRIPTION
**Pull request by akezeth**
- Used colors, typography and layout from Cindy Shin's template.
- Used mobile-first approach.
- Added 'Featured Speakers' sections dynamically using JavaScript DOM.

### CODE REVIEW

File: css/speakers-style.css [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+  margin: 0 1rem;
+  padding: 0.5rem;
+  width: 95%;
+}
+
+.less {
+  margin: 0 1rem;
+  padding: 0.5rem;
+  width: 95%;
+}
```
Comment (alexpuente0) [inline css/speakers-style.css:98]: - [ ] Please make sure all elements are matching the template's design (Show More and Less Button).
Template design: image
Your design: image

**Review by alexpuente0 — CHANGES_REQUESTED** [review]
Hello, @akezeth  



You've done a great job so far!

In this case, however, there are still some issues that you need to work on to finish this project and advance to the next one, but you are almost there!

## Highlights
- Your project looks amazing, it is almost 100% complete. Congratulations!

## Required Changes ♻️
- You seem to have missed the video part, as I couldn't find it anywhere. Please make sure you attach the video link in the Readme file and in the pull request details.
- Please make sure all elements match the design provided in the template. I left a comment letting you know which element specifically needs a makeover.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by akezeth** [comment]
- Adjusted the 'More and Less' button.
- The walkthrough video was already included in the README.md file.
image

**Review by sumairq — APPROVED** [review]
### Hi @akezeth , 



In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**